core/templates/dev/head/components/social_buttons/social_buttons_directive.html

```diff
@@ -6,60 +6,59 @@
       </h4>
     </div>
     
-    <div class="oppia-footer-icons">
+    <div class="oppia-footer-social-icons">
+      <div class="oppia-googleplus-follow">
+        <a href="https://plus.google.com/109898456505810251700/about" target="_blank"><i class="oppia-follow-icons fa fa-google-plus"></i>
+          <span class="oppia-icon-accessibility-label">Google Plus</span>
+        </a>
+      </div>
 
-        <div class="oppia-googleplus-follow">
-            <a href="https://plus.google.com/109898456505810251700/about" target="_blank"><i class="oppia-follow-icons fa fa-google-plus"></i>
-            <span class="oppia-icon-accessibility-label">Google Plus</span>
-            </a>
-        </div>
+      <div class="oppia-youtube-follow">
+        <a href="https://www.youtube.com/channel/UC5c1G7BNDCfv1rczcBp9FPw" target="_blank"><i class="oppia-follow-icons fa fa-youtube-play"></i>
+          <span class="oppia-icon-accessibility-label">Youtube</span>
+        </a>
+      </div>
 
-         <div class="oppia-youtube-follow">
-           <a href="https://www.youtube.com/channel/UC5c1G7BNDCfv1rczcBp9FPw" target="_blank"><i class="oppia-follow-icons fa fa-youtube-play"></i>
-            <span class="oppia-icon-accessibility-label">Youtube</span>
-          </a>
-        </div>
-
-        <div class="oppia-facebook-follow">
-            <a href="https://www.facebook.com/oppiaorg" target="_blank"><i class="oppia-follow-icons fa fa-facebook"></i>
-              <span class="oppia-icon-accessibility-label">Facebook</span>
-            </a>
-        </div>
+      <div class="oppia-facebook-follow">
+        <a href="https://www.facebook.com/oppiaorg" target="_blank"><i class="oppia-follow-icons fa fa-facebook"></i>
+            <span class="oppia-icon-accessibility-label">Facebook</span>
+        </a>
+      </div>
         
-        <div class="oppia-twitter-follow">
-            <a href="https://twitter.com/oppiaorg" target="_blank"><i class="oppia-follow-icons fa fa-twitter"></i>
-                <span class="oppia-icon-accessibility-label">Twitter</span>
-            </a>
-        </div>
+      <div class="oppia-twitter-follow">
+        <a href="https://twitter.com/oppiaorg" target="_blank"><i class="oppia-follow-icons fa fa-twitter"></i>
+            <span class="oppia-icon-accessibility-label">Twitter</span>
```

Comment:
Indentation here is still incorrect; ditto above.

Also, I suggest moving the "i" tag to its own line, above this one. Ditto in various places above.